client/thin-replica-client/src/thin_replica_client.cpp

```diff
@@ -202,8 +202,8 @@ std::pair<bool, ThinReplicaClient::SpanPtr> ThinReplicaClient::readBlock(Data& u
   SpanPtr span;
   if (update_in.has_event_group()) {
     id = update_in.event_group().id();
-    // TODO: Event Group traces
-    span.reset();
+    span = TraceContexts::CreateChildSpanFromBinary(
+        update_in.event_group().trace_context(), "trc_read_event_group", {}, logger_);
```

Comment:
Maybe I am missing something, why are we not passing `std::to_string(id)` as cid in this call to `CreateChildSpanFromBinary`?